charts/ingress-nginx/templates/admission-webhooks/job-patch/job-createSecret.yaml

```diff
@@ -7,6 +7,9 @@ metadata:
   annotations:
     "helm.sh/hook": pre-install,pre-upgrade
     "helm.sh/hook-delete-policy": before-hook-creation,hook-succeeded
+    {{- with .Values.controller.admissionWebhooks.annotations }}
```

Comment:
Is this already existent in values.yaml as an example? If not, can you please add there (even commented, but it is good so people may know this exists!)

Thanks!

Reply:
> 

Yeah that's make sense. Let me add a commented example into values.yaml